apps/site/assets/ts/schedule/__tests__/ScheduleTableTest.tsx

```diff
@@ -1,18 +1,14 @@
 import React from "react";
 import { mount } from "enzyme";
-import serviceData from "./serviceData.json";
-import crServiceData from "./crServiceData.json";
 import ScheduleTable from "../components/schedule-finder/ScheduleTable";
-import { ServiceScheduleInfo } from "../components/__schedule.js";
+import { busSchedule742, crSchedule } from "./ScheduleFinderTest";
```

Comment:
I wanted that export/unknown assignment to only need to happen once, rather than in multiple tests, so I centralized it here, as that is where the original fixture was defined. A case of not questioning why this data was here in the first place. I still prefer not to redefine these in multiple files, so I could create a simple `.tsx` file that simply handles the initial assignment, instead of grouping it into the `ScheduleFinderTest` if that works for you.

Reply:
I see — so if it were not for the type issue, the import/export would not be needed? I'd agree that if it _is_ needed, putting the import/export in a file dedicated to this purpose would be an improvement.

Reply:
If we're re-using these then let's have a separate file to contain them, and import that.